Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStoreConfiguration.cs

```diff
@@ -203,32 +205,87 @@ public DistributedCentralStoreConfiguration(AbsolutePath cacheRoot)
         public int MaxSimultaneousCopies { get; set; } = 10;
     }
 
+    /// <summary>
+    /// Provides Azure Storage authentication options for <see cref="BlobCentralStorage"/>
+    /// </summary>
+    public class AzureBlobStorageCredentials

```

Comment:
As I suggested, it is actually possible that this class would have just one public method: CreateCloudBlobContainer and two constructors.